install-deps.sh

```diff
@@ -66,7 +66,7 @@ function munge_debian_control {
     fi
     if $with_jaeger; then
 	sed -i -e 's/^# Jaeger[[:space:]]//g' $control
-	sed -i -e 's/^# Crimson      libyaml-cpp-dev,/d' $control
+	sed -i -e 's/^# Crimson      libyaml-cpp-dev/               libyaml-cpp-dev/g' $control
```

Comment:
@clwluvw why hardcode the number of spaces here?

Reply:
@tchaikov For keeping the indent in the `debian/control` file as `for_make_check` does in next if.

Reply:
i mean `^# Crimson      libyaml-cpp-dev`, you are trying to match the exact number of spaces with the ones in `debian/control`.

Reply:
@tchaikov If I'm not wrong you mean to change it to:
```bash
sed -i -e 's/^# Crimson libyaml-cpp-dev/               libyaml-cpp-dev/g' $contro
```

I test it locally and it's not working!

Reply:
sorry for the confusion. i'd suggest
```sh
sed -i -e 's/^# Crimson[[:space:]]\+libyaml-cpp-dev/libyaml-cpp-dev/g' $control
```

Reply:
what i don't like is the fixed number of spaces in "s/^# Crimson      libyaml-cpp-dev/".

Reply:
I don't test it yet but will it be parsable with this indentation?
```
 # Make-Check   libxmlsec1-nss,
 # Make-Check   libxmlsec1-openssl,
 # Make-Check   libxmlsec1-dev,
libyaml-cpp-dev,
                lsb-release,
 # Jaeger       nlohmann-json-dev | nlohmann-json3-dev,
                parted,
```

Reply:
presumably, yes. but I'd test it if I were you.